package/gluon-core/files/lib/gluon/domain_changed.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-domain=$(uci get gluon.@system[0].domain)
+domain=$(uci get gluon.system.domain)
```

Comment:
up until this change `/etc/config/gluon` looked like:

```
config system
    option 'domain' 'default'
```

What should it look like now?

Reply:
The answer is

```
config system system
    option 'domain' 'default'
```

Reply:
yes. thanks